### ISSUE TITLE
[Usable] Name property does not properly define for 'mm/dd/yyyy' editable text box  while navigate in scan mode.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1362]
**Actual Result:**
When focus lands on the 'mm/dd/yyyy' editable text box, Narrator is reading same information twice
Name property does not properly define for 'mm/dd/yyyy' editable text box  while navigate in scan mode.

**Note:**
Same issue also reproducible for text 'I am going to send 5 messages at different time' after activating button of 'Timestamp grouping'.

**Expected Result:**
Name property should be properly define for 'mm/dd/yyyy' editable text box  while navigate in scan mode.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/ ) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show Restaurant information using Adaptive card " button by using Tab key and press Enter on it.
5. New data will upload, Navigate to Call Restaurant.
6. New data will upload.
7. Open Narrator with Scan mode (Caps + Space key)
8. Navigate to 'mm/dd/yyyy' editable text box by using Tab key.
9. Observe that Name property does not properly define for 'mm/dd/yyyy' editable text box.

**User Impact:**
If Name property is not defined properly for control then the users will get confuse about the purpose of that control.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Usable_Reading Twice in scan mode..zip](https://github.com/Microsoft/BotFramework-WebChat/files/2584695/Usable_Reading.Twice.in.scan.mode.zip)

**Comment by compulim** [comment]
This could be a bug on Adaptive Cards project.

- [ ] Checks if `name` attribute (or similar) can be found in [this schema](https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/card-schema#inputdate)
   - [ ] If yes, implement it on MockBot and see if it works as a resolution
      - [ ] If it work, resolve this bug
      - [ ] If it doesn't work, this bug should belongs to Adaptive Cards team
   - [ ] If no, this bug should belongs to Adaptive Cards team

**Comment by corinagum** [comment]
Need to investigate

**Comment by corinagum** [comment]
Related to #1345; needs investigation

**Comment by corinagum** [comment]
> To repro this on Mock Bot, run the command `card restaurant` and click the "Call restaurant" button. Then run narrator on the input (date picker) that appears.

The Adaptive Card schema does have the property `placeholder`. However, HTML5 inputs with the type of date do not support placeholders. 

For example, [input>type=date demo on w3Schools](https://www.w3schools.com/TAGS/tryit.asp?filename=tryhtml5_input_type_date) 
1. Navigate to this example in Edge
1. Add a placeholder to the input
1. Run Narrator

"mm/dd/yyyy" is read out regardless of the placeholder, name or id. The input.date in the Web Chat Adaptive Card functions the same way.

One possible workaround for MockBot would be to provide a default value (for example, the current date), which is a property available in the Adaptive Cards Schema, but seeing as the current Adaptive Card input.date functions the same way as the HTML5 input.date, I don't think that this is a solution that addresses the accessibility of the input.

@cwhitten My current stance is that this is not an issue for Web Chat or MockBot to fix. We could forward this to Adaptive cards, since they have `placeholder` as a property when it doesn't do anything. Another way to improve the accessibility of the input would be to allow the aria-label to be added to the input. Otherwise, I believe this is an Edge bug. Do you have further input?

Note: I was not able to repro 'mm/dd/yyyy' being read twice, either in scan mode or scan off.

**Comment by corinagum** [comment]
I accidentally linked the wrong issues in my PR called Markdown cards layout. 

However, after discussing with @compulim we've decided that this should be considered as a feature request to Adaptive Cards
- remove placeholder on Input.Date
- add aria-label to Input.Date

So I will close this issue. Sorry for the confusion.

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable
